src/main/java/org/amahi/anywhere/adapter/ServerFilesAdapter.java

```diff
@@ -104,8 +109,28 @@ private View newView(ViewGroup container) {
 	private void bindView(ServerFile file, View view) {
 		ImageView fileIconView = (ImageView) view.findViewById(R.id.icon);
 		TextView fileTextView = (TextView) view.findViewById(R.id.text);
+		TextView fileSize = (TextView) view.findViewById(R.id.file_size);
+		TextView fileLastModified = (TextView) view.findViewById(R.id.last_modified);
+               LinearLayout moreInfo = (LinearLayout) view.findViewById(R.id.more_info);
 
 		fileTextView.setText(getFileName(file));
+
+               int size=getFileSize(file);
+
+               if(size==0){
+                 moreInfo.setVisibility(View.GONE);
```

Comment:
This results in a minor bug. I am not sure if I should call this a bug or not!. Well if the file size is 0 (that is in case of a newly created file). It hides the `date` also. 
I think you must not hide it, instead display both date and size. The size can be displayed as `0 Bytes`.